clap_derive/Cargo.toml

```diff
@@ -1,13 +1,10 @@
+cargo-features = ["workspace-inheritance"]
+
 [package]
 name = "clap_derive"
-version = "3.1.18"
-edition = "2018"
-include = [
-	"src/**/*",
-	"Cargo.toml",
-	"LICENSE-*",
-	"README.md"
-]
```

Comment:
FYI Our includes are meant to be the same but have diverged over time.  Now we'll be able to keep them the same!